xbmc/addons/kodi-addon-dev-kit/Makefile

```diff
@@ -0,0 +1,30 @@
+ifeq ($(CMAKE),)
+  CMAKE = cmake
+endif
+
+ifeq ($(PREFIX),)
+  PREFIX = /usr/local
+endif
+
+all: .create-kit
+
+.cmake-kit:
+	rm -rf build; mkdir -p build
+	cd build; $(CMAKE) -DCMAKE_INSTALL_PREFIX=$(PREFIX) ..
+	touch $@
+
+.create-kit: .cmake-kit
+	$(MAKE) -C build
+	touch $@
+
+.install-kit: .create-kit
+	$(MAKE) -C build install
+
+install: .install-kit
+
+clean:
+	rm -f .create-kit
+	$(MAKE) -C build clean
+
+distclean:
+	rm -rf build .create-kit .cmake-kit
```

Comment:
unrelated to this line, in lack of a better place to put it:
This Makefile will be overwritten by cmake if someone ever manually runs cmake in here without creating a build dir